pkgs/by-name/tp/tpi/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage {
+  pname = "tpi";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "turing-machines";
+    repo = "tpi";
+    rev = "v1.0.6";
+    sha256 = "1ax43dbyy41awf7s5x6kcx96dwjl3d2iir1l3qdqlbw9g1ps8jmf";
```

Comment:
```suggestion
    hash = "sha256-rkqkb3iJL4obHjTkGEUbVPJmUmfT9KKP4yoQ71cbpKs=";
```

Reply:
Is this the preferred way?

Reply:
This is the correct way.

Reply:
Thank you for clarifying. I will update it